apps/sms/views/conversation/js/conversation.js

```diff
@@ -725,16 +724,15 @@ var ConversationView = {
 
   // recalling draft for composer only
   // Bug 1010216 might use it for thread drafts too
-  handleDraft: function conv_handleDraft(threadId) {
+  handleDraft: function conv_handleDraft(draftId) {
     // TODO Bug 1010216: remove this.draft
```

Comment:
Yep, but I'm not sure if we want to get rid of it entirely - it still looks reasonable to have at least ```this.draft``` or ```this.draftId``` to have access to the current draft. Anyway we can revisit it in https://bugzilla.mozilla.org/show_bug.cgi?id=1164435